packages/design-system/components/Button.tsx

```diff
@@ -1,34 +1,41 @@
-import styled from '@emotion/styled';
-import * as React from "react";
+import styled from "@emotion/styled";
+import React, { CSSProperties } from "react";
 import theme from "../theme";
 
 export enum ButtonHierarchy {
-    Primary = "Primary",
-    Secondary = "Secondary"
+  Primary = "Primary",
+  Secondary = "Secondary",
 }
 
 export interface ButtonProps {
-    hierarchy: ButtonHierarchy;
-    children: React.ReactNode;
-    full?: boolean;
-    disabled?: boolean;
-    onClick?: (e: React.MouseEvent<HTMLButtonElement>) => unknown;
+  hierarchy: ButtonHierarchy;
+  children: React.ReactNode;
+  margin?: CSSProperties["margin"];
+  width?: CSSProperties["width"];
+  full?: boolean;
```

Comment:
fullWidth 요렇게 주는게 더 명확할듯!?

```suggestion
  fullWidth?: boolean;
```

Reply:
[4fda650](https://github.com/mash-up-kr/zuzu_web/pull/47/commits/4fda6508912ae104447a3d5d6299e743d16b38be) 반영했습니당 !